backend/corpora/common/utils/aws.py

```diff
@@ -89,3 +89,13 @@ def _restore(self):
     def _debug(self, message):
         if self.debug_logging:
             print(message)
+
+
+def delete_many_from_s3(bucket_name, dataset_uuid) -> None:
+    """
+    This deletes everything with a specific prefix from the given bucket
+
+    """
+    s3 = boto3.resource("s3")
+    bucket = s3.Bucket(bucket_name)
+    bucket.objects.filter(Prefix=f"{dataset_uuid}/").delete()
```

Comment:
We should check the inputs here, and verify that dataset_uuid is not an empty string.

https://developers.slashdot.org/story/16/04/14/1542246/man-deletes-his-entire-company-with-one-line-of-bad-code